aiidalab/app.py

```diff
@@ -248,7 +250,7 @@ def _move_to_trash(self) -> Path | None:
         trash_path = Path.home().joinpath(".trash", f"{self.name}-{uuid4()!s}")
         if self.path.exists():
             trash_path.parent.mkdir(parents=True, exist_ok=True)
-            self.path.rename(trash_path)
```

Comment:
This would fail with `OSError`  if the two directories were on different filesystems. (such as `/home/` and `/tmp`). `shutil.move` handles this case automatically.